src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -1,14 +1,13 @@
-import { CertifyIF } from '../stepper-interfaces/ReviewConfirm/certify-interface'
-import { DefineCompanyIF } from '../stepper-interfaces/DefineCompany/define-company-interface'
+import { CertifyIF, DefineCompanyIF, TombStoneIF } from '@/interfaces'
 
 // State model example
 export interface StateModelIF {
-  stateText: string | undefined
-  currentDate: string | null
-  certifyState: CertifyIF
-  currentStep: number
-  isSaving: boolean
-  isSavingResuming: boolean
-  isFilingPaying: boolean
-  defineCompanyStep: DefineCompanyIF
+  tombstone: TombStoneIF,
```

Comment:
Commas? Semicolons? Nothing? I just had this conversation with the Relationships Team and they prefer no punctuation. Should we be consistent with that?

Reply:
ok, no problem at all. I'm definitely fine with that 👍